pom.xml

```diff
@@ -491,13 +493,46 @@
                         <xmlOutput>true</xmlOutput>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.owasp</groupId>
+                    <artifactId>dependency-check-maven</artifactId>
+                    <version>${version.plugin.dependency-check}</version>
+                    <configuration>
+                        <skip>${dependency-check.skip}</skip>
+                        <skipProvidedScope>true</skipProvidedScope>
```

Comment:
I think both provided and runtime scopes are valid, as these are still helidon dependencies.

Reply:
Fixed.